user-storage-simple/README.md

```diff
@@ -39,13 +39,30 @@ All you need to build this project is Java 8.0 (Java SDK 1.8) or later and Maven
 Build and Deploy the Quickstart
 -------------------------------
 
+### Deploy on Wildfly based 
+
 To deploy this provider you must have <span>Keycloak</span> running in standalone or standalone-ha mode. Then type the follow maven command:
 
    ````
    mvn clean install wildfly:deploy
    ````
 If you want to play with and modify the example, simply rerun the maven deploy command above and the new version will be hot deployed.
 
+### Deploy on Quarkus based 
+
+To deploy this provider you must have <span>Keycloak</span> running with:
+
+```
+./kc.sh start-dev
+```
+
+Then type the follow maven command:
+
+   ````
+   mvn clean install
+   ````
+Finally copy the `.jar` file into `/providers` folder on <span>Keycloak</span>
+
```

Comment:
@pedro-hos I've just realized hot deploying doesn't work with quarkus based dist. Therefore this section has to be updated in a way that first has to be jar copied and then server could be started. 

Sorry for late response. 

I'd even suggest to remove Wildfly part all together.

Reply:
I agree! Also, I am wondering if I need to remove the following as well:

~~~
Integration test of the Quickstart
----------------------------------

1. Make sure you have an Keycloak server running with an admin user in the `master` realm or use the provided docker image
2. You need to have Chrome browser installed and updated to the latest version.
3. Run `mvn test -Pkeycloak-remote`
~~~

Reply:
It'd probably be good to update it as well, together with checking the test as well.  

I need to find out current plans regarding the tests. There is: https://github.com/keycloak/keycloak-quickstarts/blob/69556a965568e68fb85e125b497262b2d56df074/scripts/run-tests.sh#L85-L92

Let me check it and get back to you once I know more, thanks for the patience.